lib/tasks/rec.rake

```diff
@@ -37,16 +37,14 @@ namespace :alef do
       # Najde zelane opdorucace
       activity_rec = Recommender.where(name: 'Activity').take
       naive_activity_rec = Recommender.where(name: 'NaiveActivity').take
-      naive_concept_rec = Recommender.where(name: 'NaiveConcept').take
+      naive_concept_rec = Recommender.where(name: 'NaiveConcInvolvedIept').take
+      irt_rec = Recommender.where(name: 'Irt').take
 
 
       # Nastavi ich vahu pre jednotlive odporucania
-      RecommendersOption.create(recommendation_configuration_id: conf_default.id, recommender_id: naive_activity_rec.id, weight: 10)
-      RecommendersOption.create(recommendation_configuration_id: conf_default.id, recommender_id: naive_concept_rec.id, weight: 3)
+      RecommendersOption.create(recommendation_configuration_id: conf_default.id, recommender_id: naive_activity_rec.id, weight: 2)
+      RecommendersOption.create(recommendation_configuration_id: conf_default.id, recommender_id: irt_rec.id, weight: 1)
 
-      RecommendersOption.create(recommendation_configuration_id: conf_alter.id, recommender_id: naive_activity_rec.id, weight: 10)
```

Comment:
Pozeram, ze v novom nastaveni recommenderov sa nastavuje IRT pre konfiguraciu `default` a konfiguracia `alternative` sa nenastavuje vobec.
Pozor, podla https://github.com/PewePro/alef-tng/blob/master/lib/recommender_system/hybrid_recommender.rb#L16 sa pre pouzivatelov s nastavenou skupinou `B` pouziva alternativna konfiguracia (ktora je po novom bez odporucacov). Rozdelenie A/B sa robilo minuly rok pre odporucace, novi pouzivatelia dostavaju default skupinu `X`, ktora dostane default (s irt) (rovnako ako A).

Pre info:
`SELECT "group",COUNT(*),MAX(created_at) FROM users GROUP BY "group" ORDER BY "group";`
```
A	123	2015-05-30 12:58:44.119831
B	120	2015-05-30 12:58:44.130195
X	270	2016-03-21 13:36:28.735549
```

Technicky by to, ze niektori nedostanu novy odporucac, nemuselo vadit, lebo minulorocnych studentov so skupinou `B` sem uz tento semester vela nechodi:
`SELECT "group",COUNT(*) FROM users WHERE last_sign_in_at > '2016-02-14' GROUP BY "group" ORDER BY "group";`
```
A	6
B	7
X	239
```

ALE, ako sa to zachova, ked sem pridu a dostanu taku RecommendationConfiguration, ktora nema ziadne odporucace? Treba to vyskusat a spravit akciu:
- vytvorit rovnako nastavenu konfiguraciu `alternative` s NaiveActivity a Irt odporucacmi
alebo
- zmenit ten kod v `hybrid_recommender.rb`, nech teraz dava vsetkym default bez ohladu na skupinu
alebo
- zmenit nastavenie skupin pouzivatelom v DB (najmenej preferovane riesenie, ked uz to tam mame ulozene, ako boli rozdeleni, tak to tam nechajme, aj pre to, keby niekto analyzoval/v odporucaci/kdekolvek pouzival ich aktivitu, aby mal informaciu cim sa im odporucalo, lebo mohli robit ine cesty po LO)